packages/autorest.go/src/generator/operations.ts

```diff
@@ -576,7 +576,7 @@ function createProtocolRequest(group: OperationGroup, op: Operation, imports: Im
     // add query parameters
     const encodedParams = new Array<Parameter>();
     const unencodedParams = new Array<Parameter>();
-    for (const qp of values(aggregateParameters(op)).where((each: Parameter) => { return each.protocol.http !== undefined && each.protocol.http!.in === 'query'; })) {
+    for (const qp of values(aggregateParameters(op)).where((each: Parameter) => { return each.protocol.http !== undefined && each.protocol.http?.in === 'query'; })) {
```

Comment:
We used to explicitly disallow null, now we will allow it.  If that was a bug why didn't we ever hit it?  Is this a gap in our test suite?

Reply:
It's because all parameters from REST participate in some portion of the HTTP request, so `http` was never `null`.  Now that `ResumeToken` is a real parameter, and it doesn't go in any portion of the HTTP request, we must use the conditional operator.